Topic 1/src/com/globant/topic1/exercise1/strategy/CashStrategy.java

```diff
@@ -0,0 +1,11 @@
+package com.globant.topic1.exercise1.strategy;
+
+// Concrete Strategy
+public class CashStrategy implements IPaymentMethodStrategy {
```

Comment:
Just a bit correction here.
The classes that implement an interface, need ti have __Impl in the name or put it in an __impl package.